components/search-page/StyledCollectiveCard.js

```diff
@@ -219,7 +219,7 @@ const StyledCollectiveCard = ({
                   .filter(tag => !IGNORED_TAGS.includes(tag))
                   .slice(0, 4)
                   .map(tag => (
-                    <StyledTag key={tag} display="inline-block" variant="rounded-right" m={1}>
+                    <StyledTag key={tag} display="inline-block" variant="truncated" m={1}>
```

Comment:
Good call to address this with a new prop. However, this `truncated` variant uses the `rounded-right` variant styles, and arguable we'll want to re-use this truncation feature with other tag styles in the future. So why not use a new prop:

```suggestion
                    <StyledTag key={tag} display="inline-block" variant="rounded-right" truncateAt="10em" m={1}>
```